views/house.erb

```diff
@@ -0,0 +1,58 @@
+<div class="page-section page-section--grey text-center">
+  <div class="container">
+    <h3>We’re cataloguing every politician in <%= @page.country.name %>.</h3>
+    <p>We currently have information for…</p>
+  </div>
+</div>
+
+<div class="page-section">
+  <div class="container">
+
+    <div class="country__legislature">
+      <div class="country__legislature__header">
+        <h3><%= @page.house.name %></h3>
```

Comment:
This seems a little redundant now that there can only be one legislature on the page, and it's already highlighted in the header.